lxc.md

```diff
@@ -123,7 +123,6 @@ Set up hostfiles on each box with hardcoded IP addresses
 
 
 ```
-127.0.0.1 localhost n1 n1.local
```

Comment:
Ohhh hang on, I wonder if your notes aren't listening on loopback, and they're trying to initiate a connection back to their own hostname.